docs/GettingStarted.asciidoc

```diff
@@ -437,7 +437,7 @@ For Fedora, a sample run might be:
 [source,sh]
 --------------------------------------------------------------------------------
 # Run the first test
-openqa-cli api -X POST isos \
+openqa-cli api -X POST isos async=0 \
```

Comment:
As these seem to be the first and most simple examples I would just stick with the default which for us is async=0 but not explicitly stating it. After this example there is already a reference to the users guide for "more details" so that should ok